server/src/api/site.rs

```diff
@@ -415,12 +419,29 @@ impl Perform for Oper<GetSite> {
       0
     };
 
+    // Giving back your user, if you're logged in
+    let my_user: Option<User_> = match &data.auth {
+      Some(auth) => match Claims::decode(&auth) {
+        Ok(claims) => {
+          let user_id = claims.claims.id;
+          let mut user = blocking(pool, move |conn| User_::read(conn, user_id)).await??;
+          user.password_encrypted = "".to_string();
+          user.private_key = None;
+          user.public_key = None;
+          Some(user)
+        }
+        Err(_e) => None,
+      },
+      None => None,
+    };
```

Comment:
Might be better to have a separate API endpoint for the user info. It doesnt seem related to the site info.

Reply:
I thought about that, but it'd take a LOT of front end changes, because every single component would need to pull that in addition to `GetSite`... they already have to get the site info anyway, so it seems like a good place to have it. Plus since its an Option anyway, the front end would have to add additional logic to not even make the call since it doesn't have an auth. This is just a much easier way to do it, and not have additional calls.